API/DTOs/TournamentRequestQueryDTO.cs

```diff
@@ -30,6 +31,49 @@ public class TournamentRequestQueryDTO : IPaginated
     [EnumDataType(typeof(Ruleset))]
     public Ruleset? Ruleset { get; init; }
 
+    /// <summary>
+    /// Filters results for only tournaments with a partially matching name or abbreviation (case insensitive)
+    /// </summary>
+    public string? Name { get; init; }
```

Comment:
Would `SearchKey` be better here since it covers Name and Abbreviation?

Reply:
`SearchQuery` maybe more inclusive

Reply:
Sure